src/styles/bookInstance.css

```diff
@@ -0,0 +1,152 @@
+.SinglebookArrange {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  margin: 0;
+  padding: 15px;
+  padding-left: 20px;
+  background-color: white;
+  flex-wrap: wrap;
+}
+
+.bookImfo span {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  opacity: 0.5;
+}
+
+.bookImfo h2 {
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: 0;
+}
+
+.bookImfo h3 {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+.bookImfo button {
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  background-color: transparent;
+  border: 0;
+  border-right: 3px solid #e8e8e8 !important;
+  cursor: pointer;
```

Comment:
- [ ] Kindly it will be better 😃 if you remove the ```!important``` from the style here as this is from basic CSS bad practices are not used, so The ```!important``` rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. The ```!important``` rule💯 prevails over all the previous styles and says, “I’m more important, forget about all of that other CSS and use me!“. Using this out of laziness will give you a headache later on as your style sheet will be harder to maintain.🚀👌You can read [Here](https://speckyboy.com/good-bad-css-practices) about best practices 😊💯